src/GenFacades.Core.Desktop/GenFacades.Core.Desktop.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
Regarding having just one project: we could have GenFacades.exe expose the functionality publically and use that from the build task. It's a bit awkward in my opinion, though, and doesn't feel very clean (the console app part would be unused, the build task calling a method from an .exe...). This feels like the more proper thing to do.